atlasdb-config/src/main/java/com/palantir/atlasdb/config/AtlasDbConfig.java

```diff
@@ -48,6 +49,8 @@ public abstract class AtlasDbConfig {
 
     public abstract KeyValueServiceConfig keyValueService();
 
+    public abstract Optional<TransactionKeyValueServiceConfig> transactionKeyValueService();
```

Comment:
Can we annotate `@Beta` or similar just so I know this can be changed later on without too much friction?